Sources/ArcGISToolkit/Components/Offline/PreplannedListItemView.swift

```diff
@@ -17,11 +17,14 @@ import ArcGIS
 
 @MainActor
 @preconcurrency
-public struct PreplannedListItemView: View {
+struct PreplannedListItemView: View {
     /// The view model for the preplanned map.
     @ObservedObject var model: PreplannedMapModel
     
-    public var body: some View {
+    /// The closure to perform when the map selection changes.
+    let onMapSelectionChanged: ((Map) -> Void)
```

Comment:
Without the question mark, the outer parentheses are no longer needed.

```suggestion
    let onMapSelectionChanged: (Map) -> Void
```